policy_packs/aws/vpc/enforce_block_public_access_for_security_groups/README.md

```diff
@@ -0,0 +1,103 @@
+---
+categories: ["security"]
+---
+
+# Enforce Block Public Access for AWS VPC Security Groups
+
+Enforcing Block Public Access for AWS VPC security groups is essential to prevent unauthorized access and exposure of sensitive data to the public internet. This measure ensures that ingress rules for ports 22 (SSH) and -1 (all ports) from 0.0.0.0/0 and ::/0 are blocked, minimizing the risk of security breaches and ensuring compliance with security best practices and regulatory requirements.
+
+This [policy pack](https://turbot.com/guardrails/docs/concepts/resources/smart-folders) can help you configure the following settings for VPC security groups:
+
+- Enforce Block Public Access, which would prevent unauthorized access
+- Set individual Block Public Access settings for VPC security groups
```

Comment:
- Reject and revoke unapproved rules that allow ingress for ports 22 (SSH) and -1 (all ports) from 0.0.0.0/0 and ::/0.